openslides_backend/models/models.py

```diff
@@ -3,7 +3,7 @@
 from openslides_backend.models import fields
 from openslides_backend.models.base import Model
 
-MODELS_YML_CHECKSUM = "c225b5b775784fb39a8a867de8b6f177"
+MODELS_YML_CHECKSUM = "b314ac93c0250fd96d438686dea8c0eb"
```

Comment:
Missing field vote_decrypt_public_main_key

Reply:
No. The field is calculated. It is not added to the models.py. See https://github.com/OpenSlides/openslides-backend/blob/997bc2d35e4ac4bfee37b94bdd21db4f7b169434/cli/generate_models.py#L202-L203